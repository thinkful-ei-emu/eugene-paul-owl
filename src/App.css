.App {
  text-align: center;
}

.container{
  height:100vh;
}
.control-bar{
  display:block;
  height:75px;
}

.chatter-box{
  display:inline-block;
  width:330px;
  height:calc(100% - 75px);
  overflow:auto;
  vertical-align:top;
}

.stage{
  display:inline-block;
  width:calc(100% - 330px);
  height:calc(100% - 75px);
  vertical-align:top;
}

