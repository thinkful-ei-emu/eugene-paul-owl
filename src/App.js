import React from 'react';
import './App.css';
import ChatLog from './ChatLog';
import PartList from './PartList';
import Stage from './Stage';

function App(props) {
  return (
    <div className="container">
      <div className="chatter-box">
        <div className='chat-head'>

        </div>
        <div className='chat-results'>
          <ChatLog chatEvents={props.chatEvents} participants={props.participants}/>
          {<PartList participants={props.participants}/>}
        </div>
      </div>
      <div className="stage">
        {<Stage/>}
      </div>
    <div className="control-bar">
      
    </div>
    
    </div>
    
  );
}

export default App;
