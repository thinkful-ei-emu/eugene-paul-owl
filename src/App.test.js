import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';
import {chatEvents, participants} from './Store';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App chatEvents={chatEvents} participants={participants}/>, div);
  ReactDOM.unmountComponentAtNode(div);
});


it('renders UI with default props', () => {
  const tree = renderer.create(<App chatEvents={chatEvents} participants={participants}/>);
  expect(tree).toMatchSnapshot();
});