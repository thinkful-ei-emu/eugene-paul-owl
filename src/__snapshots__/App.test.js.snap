// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UI with default props 1`] = `
<div
  className="container"
>
  <div
    className="chatter-box"
  >
    <div
      className="chat-head"
    />
    <div
      className="chat-results"
    >
      <div>
        <div>
          <div
            className="message"
          >
            <img
              alt="avatar"
              className="user-avatar message-avatar"
              src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
            />
            <div
              className="message-body"
            >
              <h6
                className="message-body-header"
              >
                Axe Kubicka
              </h6>
              <span
                className="message-body-time"
              >
                 
                07:48:04 GMT-0500 (Eastern Standard Time)
                 
              </span>
            </div>
            <div
              className="message-body-content"
            >
              <p>
                Hello world
              </p>
            </div>
          </div>
          <div
            classNAme="system-message"
          >
            <strong>
              Caty Flucker
            </strong>
          </div>
          <div
            classNAme="system-message"
          >
            <strong>
              Frank Runciman
            </strong>
          </div>
          <div
            classNAme="system-message"
          >
            <strong>
              Koren Templeton
            </strong>
          </div>
          <div
            classNAme="system-message"
          >
            <strong>
              Koren Templeton
            </strong>
          </div>
          <div
            classNAme="system-message"
          >
            <strong>
              Koren Templeton
            </strong>
          </div>
          <div
            classNAme="system-message"
          >
            <strong>
              Ashla Attwool
            </strong>
          </div>
          <div
            classNAme="system-message"
          >
            <strong>
              Axe Kubicka
            </strong>
          </div>
          <div
            classNAme="system-message"
          >
            <strong>
              Axe Kubicka
            </strong>
          </div>
        </div>
        <form
          className="chat-form"
        >
          <textarea
            className="chat-form-input"
            placeholder="Chat"
            style={
              Object {
                "height": "45px",
              }
            }
          />
          <input
            className="chat-form-submit"
            type="submit"
          />
        </form>
      </div>
      <div>
        <div
          className="participant"
        >
          <img
            alt="avatar"
            className="avatar"
            src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
          />
          <h3>
             
            Koren Templeton
          </h3>
          <p />
        </div>
        <div
          className="participant"
        >
          <img
            alt="avatar"
            className="avatar"
            src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
          />
          <h3>
             
            Caty Flucker
          </h3>
          <p />
        </div>
        <div
          className="participant"
        >
          <img
            alt="avatar"
            className="avatar"
            src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
          />
          <h3>
             
            Axe Kubicka
          </h3>
          <p />
        </div>
        <div
          className="participant"
        >
          <img
            alt="avatar"
            className="avatar"
            src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
          />
          <h3>
             
            Frank Runciman
          </h3>
          <p />
        </div>
        <div
          className="participant"
        >
          <img
            alt="avatar"
            className="avatar"
            src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
          />
          <h3>
             
            Ashla Attwool
          </h3>
          <p />
        </div>
      </div>
    </div>
  </div>
  <div
    className="stage"
  />
  <div
    className="control-bar"
  />
</div>
`;
